client/index.css

```diff
@@ -199,3 +199,119 @@ ul {
 .chat ul li.typing div {
     animation: flash 2s infinite;
 }
+#settings {
+    position: absolute;
+    top: 40px;
+    left: 0;
+    right: 0;
+    width: 345px;
+    height: auto;
+    margin: auto;
+}
+#settings #icon-username {
+    position: absolute;
```

Comment:
Reposition this without `absolute` positioning. Use appropriate margins to express your presentation semantics – you don't *mean* for the username to be at the exact location (30px, 100px). You *mean* for it to be centered in a certain manner. Your CSS should reflect that to aid maintainability.